libraries/chain/thread_utils.cpp

```diff
@@ -23,10 +24,33 @@ void named_thread_pool::start() {
    _ioc_work.emplace( boost::asio::make_work_guard( _ioc ) );
    _ioc.restart();
    for( size_t i = 0; i < _num_threads; ++i ) {
-      _thread_pool.emplace_back( [&ioc = _ioc, &name_prefix = _name_prefix, i]() {
+      _thread_pool.emplace_back( [&ioc = _ioc, &name_prefix = _name_prefix, on_except = _on_except, i]() {
```

Comment:
Why not capture `&_on_except` instead of making copies `on_except = _on_except`?